board/rt1051/README.md

```diff
@@ -0,0 +1,25 @@
+
+# board-rt1051
+
+This directory contains low-level drivers and files necessary to configure and run rt1051.
+
+## Files organisation
+
+- Linker scripts for rt1051 `ldscripts/`
+- Memory organisation
+    - Pure `puretx/`
+    - Bell `bellrx/`
+- Crashdump
+- Common Microcontroller Software Interface Standard (CMSIS)
```

Comment:
Kudos for expanding the acronym, it can be not so obvious for someone

Reply:
Thanks :)